apps/demos/Demos/CardView/CardTemplate/Angular/app/license-info/license-info.component.html

```diff
@@ -0,0 +1,13 @@
+<p
+  ><b>Image licensed under: </b> <span>{{ vehicle.LicenseName }}</span></p
+>
+<p
+  ><b>Author: </b> <span>{{ vehicle.Author }}</span></p
+>
+<p
+  ><b>Source link: </b>
+  <a [href]="vehicleLink"> {{ vehicleLink }} </a>
```

Comment:
Redundant spaces in ```a``` tag
```suggestion
  <a [href]="vehicleLink">{{ vehicleLink }}</a>
```